### ISSUE TITLE
ConcurrentModificationException when updating audits

### ISSUE THREAD

**Issue by Gamebuster19901** [Gamebuster19901/ExciteBot#43]
```
java.util.ConcurrentModificationException
    at java.base/java.util.HashMap$HashIterator.nextNode(HashMap.java:1493)
    at java.base/java.util.HashMap$EntryIterator.next(HashMap.java:1526)
    at java.base/java.util.HashMap$EntryIterator.next(HashMap.java:1524)
    at com.gamebuster19901.excite.bot.audit.Audit.updateAuditsFile(Audit.java:307)
    at com.gamebuster19901.excite.Main.main(Main.java:122)


This should not happen, submit a bug report!
```

**Comment by Gamebuster19901** [comment]
This is caused because new audits can be created from jda threads while the main thread is iterating over the audit map.

The solution is to synchronize over the map when it's accessed.

**Comment by Gamebuster19901** [comment]
Just use Collections.synchronizedSet

**Comment by Gamebuster19901** [comment]
Fixed in d50b78f

**Comment by Gamebuster19901** [comment]
Ugh I hate multi trheading
